pyclient/pymtt.py

```diff
@@ -144,87 +144,37 @@
                       help="Use when testing your MTT client setup; results that are generated and submitted to the database are marked as \"trials\" and are not included in normal reporting.")
 args = parser.parse_args()
 
+# check to see if MTT_HOME has been set - we require it
+try:
+    mtthome = os.environ['MTT_HOME']
+except KeyError:
+    print("MTT_HOME could not be found in your environment")
+    print("Python client requires that this be set and point")
+    print("to the top-level directory of your MTT installation")
+    sys.exit(1)
```

Comment:
I think it would make sense to ensure MTT_HOME is an absolute path.
I would go with forcing MTT_HOME to be an absolute path, as opposed to expanding it ourselves.
i.e
```python
if not os.path.isabs(mtthome):
    print("MTT_HOME environment variable is not an absolute path")
    sys.exit(1)
```

Reply:
Otherwise, this looks great. I think this is a good, clean solution to #673.

Reply:
good suggestion - I added it